src/main/resources/jenkins/plugins/slack/user/SlackUserProperty/config.jelly

```diff
@@ -0,0 +1,11 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:entry field="userId" title="${%User Id}">
```

Comment:
could you add help for this? saying where to get the ID from?

Reply:
I can include a link to an API method tester such as https://api.slack.com/methods/users.info/test. It's just not that simple to actually look up your own slack user Id (or at least not in an enterprise Slack workspace).